api/rs/slint/Cargo.toml

```diff
@@ -49,6 +49,10 @@ backend-gl-x11 = ["i-slint-backend-selector/backend-gl-x11", "std"]
 ## Wayland window system on Unix.
 backend-gl-wayland = ["i-slint-backend-selector/backend-gl-wayland", "std"]
 
+## If enabled, calls of `debug()` in `.slint` files use to the `debug!()` macro
+## of the `log` crate instead of just `println!()`.
+log = ["dep:log"]
```

Comment:
you actually don't need that, it is automatic.  And you can add the `##` before the `log`  dependency

Reply:
So should i move the documentation to the dependency section or should we let the feature entry commented out until we can use `dep:` and then uncomment it to have it listed as a feature explicitly?

Reply:
move the documentation to the dependency section